detox/android/detox/build.gradle

```diff
@@ -93,6 +93,16 @@ dependencies {
 // Spek (https://spekframework.org/setup-android)
 if (rootProject.hasProperty('isOfficialDetoxLib') ||
     rootProject.hasProperty('isOfficialDetoxApp')) {
+    buildscript {
+        repositories {
+            jcenter()
+        }
+
+        dependencies {
+            classpath 'de.mannodermaus.gradle.plugins:android-junit5:1.5.2.0'
+        }
+    }
+
```

Comment:
@valentynberehovyi why is this needed? it's already in the root build.gradle, were you trying to move it?

Reply:
In case of using detox as android project dependency classpath from root project is not available.

Reply:
Could you elaborate? what wasn't working specifically?

Reply:
Spek tests were not working in case of using detox as android project dependency.

Reply:
@valentynberehovyi can you explain exactly what failed (what gradle task/command) for you that required you to add this?

Reply:
Mine test app uses detox as project dependency. Also mine app set ext.isOfficialDetoxApp=true to develop test spek as well.
Without classpath in buildscript dependency gradle not able to resolve plugin **'de.mannodermaus.android-junit5'** . I'll move it to mine root project also for now.